src/wymeditor/core.js

```diff
@@ -944,6 +1022,44 @@ WYMeditor._computeBasePath = function (wymPath) {
     return wymPath.substr(0, lastSlashIndex + 1);
 };
 
+WYMeditor._getSkinUIOverrides = function (providedOptions, skin) {
+    var uiOverrides = {},
+        skinOverride;
+
+    if (!providedOptions) {
+        return uiOverrides;
+    }
+
+    if (skin.UI_COMPONENTS) {
+        jQuery.each(WYMeditor.UI_COMPONENTS, function (i, componentAttr) {
```

Comment:
We [can rely on having ES5](https://github.com/wymeditor/wymeditor#requirements) and the iterable is an array so this should be using `Array.prototype.forEach`, instead of `jQuery.each`.